client/components/Users.js

```diff
@@ -83,12 +45,29 @@ export class Users extends React.Component {
                     {
                     this.state.users.map(user => {
                         return (
-                            <tr key={++id}>
-                                <td>{user}</td>
-                                <td>{`${user}@email.com`}</td>
+                            <tr key={user.id}>
+                                <td>{user.id.slice(0,8)}</td>
+                                <td>{user.email}</td> 
+
+                                {/* Handels Resetting Password */}
                                 <td><button onClick={this.handleResetPassword}>Reset</button></td>
-                                <td><button name={user} value={id} onClick={this.handleDeleteUser}>X</button></td>
-                                <td><input name={user} type="checkbox" onChange={this.handleChecked} /></td>
+
+                                {/* Handels Deleting a User */}
+                                <td><button name={user.name} value={user.id} onClick={() => {
+                                     if(user.isAdmin === false){
```

Comment:
this doesn't work. it's checking whether the user you're clicking is an admin. the isAdmin check needs to be done on the backend to see whether the requesting user is an admin, not the deleted user.